CHANGELOG.md

```diff
@@ -19,7 +34,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Security
 
-## [2.10.2]
+## [2.10.2] - 05/07/2024
```

Comment:
Sometime in the future we should change these dates to be year/month/day to make it unambiguous and sortable.